esx_service/utils/kvESX.py

```diff
@@ -102,17 +161,33 @@ def kvESXInit():
     lib.DiskLib_DBSet.argtypes = [c_uint32, c_char_p, c_char_p]
     lib.DiskLib_DBSet.restype = c_uint32
 
-    return None
+    return
+
+def kv_esx_init():
+   global using_64bits
 
+   if sys.hexversion >= PYTHON64_VERSION:
+      kv_ESX_lib64_init()
+      using_64bits = True
+   else:
+      kv_ESX_lib_init()
+      using_64bits = False
+
+   return
 
 # Open a VMDK given its path, the VMDK is opened locked just to
 # ensure we have exclusive access and its not already in use.
 def volOpenPath(volpath):
-    dhandle = c_uint32(0)
-    ihandle = c_uint32(0)
+    if using_64bits:
+       dhandle = c_uint64(0)
```

Comment:
Can't you just have a wrapper for c_uint (get_c_uint) and call it when needed, instead of doing ' if using_64bits:' all over the place ?